src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -227,12 +236,9 @@ private CommandResult executeCommand(String commandText) throws CommandException
 
             if (commandResult.isShowHelp()) {
                 handleHelp();
-            }
-
-            if (commandResult.isExit()) {
+            } else if (commandResult.isExit()) {
```

Comment:
I think it would be better not not use else if? Best is the guard clause method.